src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -10,10 +11,11 @@ import { WithId } from "utils/id";
 import "./UserAvatar.scss";
 
 export interface UserAvatarProps {
-  user?: WithId<User>;
+  user?: WithId<User> | FirebaseReducer.Profile<User>;
   containerClassName?: string;
   imageClassName?: string;
-  isOnline?: boolean;
+  statusClassName?: string;
```

Comment:
Any usecases we would want to modify statusClassName?

Reply:
boolean can't represent all available statuses